lib/rspec/support/spec/stderr_splitter.rb

```diff
@@ -38,6 +38,10 @@ def to_io
       def write(line)
         return if line =~ %r{^\S+/gems/\S+:\d+: warning:} # http://rubular.com/r/kqeUIZOfPG
 
+        # Ruby 2.7.0 complains about hashes used in place of keyword arguments
+        # Aruba 0.14.2 uses this internally triggering that here
+        return if line =~ %r{lib/ruby/2\.7\.0/fileutils\.rb:622: warning:}
```

Comment:
@mvz you might be interested

Reply:
Thanks @pirj. This is fixed in Aruba 0.14.14.